plugins/woocommerce-admin/client/wp-admin-scripts/product-tracking/shared.ts

```diff
@@ -13,23 +13,127 @@ import { waitUntilElementIsPresent } from './utils';
  *
  * @return object
  */
+
+const isElementVisible = ( element: HTMLElement ) =>
+	! ( window.getComputedStyle( element ).display === 'none' );
+
+let initialStockValue = 0;
+
 const getProductData = () => {
-	return {
+	const isBlockEditor =
+		document.querySelectorAll( '.block-editor' ).length > 0;
+
+	let description_value = '';
+	let tagsText = '';
+
+	if ( ! isBlockEditor ) {
+		tagsText = (
+			document.querySelector(
+				'[name="tax_input[product_tag]"]'
+			) as HTMLInputElement
+		 ).value;
+		const content = document.querySelector(
+			'#content'
+		) as HTMLInputElement;
+		if ( content && isElementVisible( content ) ) {
+			description_value = content.value;
+		} else if ( typeof tinymce === 'object' && tinymce.get( 'content' ) ) {
+			description_value = tinymce.get( 'content' ).getContent();
+		}
+	} else {
+		description_value = (
+			document.querySelector(
+				'.block-editor-rich-text__editable'
+			) as HTMLInputElement
+		 )?.value;
+	}
+
+	const productData = {
 		product_id: ( document.querySelector( '#post_ID' ) as HTMLInputElement )
 			?.value,
 		product_type: (
 			document.querySelector( '#product-type' ) as HTMLInputElement
 		 )?.value,
 		is_downloadable: (
 			document.querySelector( '#_downloadable' ) as HTMLInputElement
-		 )?.value,
+		 )?.checked,
```

Comment:
`wcadmin_product_update` records this as `No` or `Yes`. We should do the same here for consistency.